dracut/40clevis-unlock/clevis-unlock.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=Unlock disks using Clevis
+DefaultDependencies=no
+# clevis-luks-generic-unlocker depends on executables in /sysusr/usr.
+RequiresMountsFor=/sysusr/usr/
+# dracut-initqueue will wait for /dev/disk/by-label/ROOT, so we must decrypt the root disk before dracut-initqueue.
```

Comment:
I think it's still good to add dependencies explicitly in addition, i.e., `initrd-root-fs.target` should depend on this (it's used by `initrd-setup-root.service` and `ignition-mount.service`.

Reply:
```suggestion
# dracut-initqueue will wait for /dev/disk/by-label/ROOT, so we must decrypt the root disk before dracut-initqueue.
Before=initrd-root-fs.target
```

Reply:
Thinking about it I guess I'm too paranoid. The unlocking is a precondition to find the rootfs and we already wait for that anyway, so things should be fine.

Reply:
> In https://github.com/flatcar/mantle/pull/493 I see that we explicitly want to support an encrypted rootfs. If that's the case we should make sure that the rootfs gets unlocked and mounted before initrd-setup-root.service. I've made a suggestion above which should work in both boot paths: the first uses Ignition, the second doesn't but uses the clevis service (If I understood that correctly).

Sounds good. I added the `Before=initrd-root-fs.target`.